model_analyzer/config/input/config_command_report.py

```diff
@@ -209,6 +209,10 @@ def set_config_values(self, args):
 
         super().set_config_values(args)
 
+    # TODO: Update this when adding support for detailed reporting
```

Comment:
Do we no longer tag these with ticket numbers?

Reply:
I've added the ticket number.